.mvn/maven.config

```diff
@@ -1,3 +1,3 @@
--Drevision=0.3.0
+-Drevision=0.4.0
 -Dsha1=
--Dchangelist=-SNAPSHOT
```

Comment:
you don't need to remove this Jenkins and maven both handle it correctly

Reply:
understood. tnx